src/sagemaker/fw_utils.py

```diff
@@ -148,7 +148,15 @@
 ]
 
 
-TORCH_DISTRIBUTED_SUPPORTED_FRAMEWORK_VERSIONS = ["1.11", "1.11.0"]
+TORCH_DISTRIBUTED_TRAINIUM_SUPPORTED_FRAMEWORK_VERSIONS = [
```

Comment:
nit: Can we call this `TRAINIUM_SUPPORTED_FRAMEWORK_VERSIONS` like `TRAINIUM_SUPPORTED_DISTRIBUTION_STRATEGIES` at the bottom? This will remove coupling between launcher and instance type here.

Reply:
modified thanks!